docs/setup/administrators/configuration.md

```diff
@@ -175,6 +175,12 @@ Staff e-mail domain. Used for OAUTH2 whitelist default value and staff account c
 
 ----
 
+Should staff identities be obscured (ie. comments will be ORG_LONG_NAME rather than "John Doe").
+
+    SUBMISSIONS_DRAFT_ACCESS_STAFF = env.bool('SUBMISSIONS_DRAFT_ACCESS_STAFF', False)
```

Comment:
`HIDE_STAFF_IDENTITY`

Reply:
gah my copy & pastes are getting me in trouble